src/app/groups/containers/user-group-invitations/user-group-invitations.component.ts

```diff
@@ -36,10 +41,12 @@ import { mapToFetchState } from '../../../utils/operators/state';
     NgSwitchDefault,
     AsyncPipe,
     NgClass,
+    JoinGroupConfirmationDialogComponent,
   ],
 })
 export class UserGroupInvitationsComponent implements OnDestroy {
   processing = false;
+  joinGroupConfirmationDialog?: JoinGroupConfirmationDialogParams;
```

Comment:
Even if you remove the `id` from the `JoinGroupConfirmationDialogParams` as discussed above, I think you should one single variable like `pendingJoinRequest` with the `id`  and params, so that you can keep your `@if (joinGroupConfirmationDialog)` condition (which would become`@if (pendingJoinRequest)`) and pass the params to the dialog component.